external/LibraryThemeStyles/src/Service/PresetManagerFactory.php

```
@@ -0,0 +1,17 @@
+
+namespace LibraryThemeStyles\Service;
+
+use Interop\Container\ContainerInterface;
+use Laminas\ServiceManager\Factory\FactoryInterface;
+
+/**
+ * Factory for PresetManager
+ */
+class PresetManagerFactory implements FactoryInterface
+{
+    public function __invoke(ContainerInterface $container, $requestedName, array $options = null): PresetManager
```

PHPMD (2.15.0)

Avoid unused parameters such as '$container'. (undefined)

(UnusedFormalParameter)